### DISCUSSION TITLE
Bitmap already exists

### DISCUSSION THREAD

**Discussion by billystaggs** [abbbi/virtnbdbackup#190] [Q&A / Support] [answered]
I am unable to run a Full or Inc backup.  A offline copy works fine.  I do not show any checkpoints.  What am I doing wrong? 

[2024-07-24 17:15:29] ERROR root virtnbdbackup - startBackupJob [MainThread]: Failed to start backup: [internal error: unable to execute QEMU command 'transaction': Bitmap already exists: virtnbdbackup.0]

root@leo:/media/TR002/vmbackups/gibson/Hourly # virsh checkpoint-list Debian01
 Name   Creation Time
-----------------------
  
Thanks,
--
bstaggs

**Comment by abbbi** [comment, marked as answer]
Pls Check past issues and discussions on this. There is an inconsistency between which bitmaps/checkpoints libvirt thinks and which ones are actually defined in the qemu disk image. Use qemu-img Tool to cleanup/Remove the bitmap and Start over

Also Check the readme Part for backup in transient/Cluster envs if your domains are migrated between hosts which can also Clause this Situation

**Reply by billystaggs** [reply]
Thanks for the reply.  The domain was not migrated in my case.  I will research the gemu-img tool.

**Reply by billystaggs** [reply]
Thanks for pointing me in the right direction.  Backups seem to be working again.  Just in case it helps someone else:
qemu-img info filename.qcow2
qemu-img bitmap --remove filename.qcow2 virtnbdbackup.0

Thanks again.

**Comment by abbbi** [comment]
ive added an FAQ for this now:

https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backup-fails-with-unable-to-execute-qemu-command-transaction-bitmap-already-exists
